README.md

```diff
@@ -41,114 +41,113 @@ cyrusauth:
 
 ## Installing ZNC
 
-If you are building from git, you will need to run `./autogen.sh` first to produce the `configure` script.
-Note that this requires `automake` and `gettext` to be installed.
+If you are building from git, you will need to run `./autogen.sh` first to
+produce the `configure` script. Note that this requires `automake` and
+`gettext` to be installed.
 
 Installation is done with the `./configure ; make ; make install` commands.
 
-You can use
-	./configure --help
-if you want to get a list of options, though the defaults should be suiting
-most needs. After you compiled it with make (or gmake if make doesn't work) you
-can install it with
-	make install
-though you don't need to as ZNC supports in-place execution.
+You can use `./configure --help` if you want to get a list of options,
+though the defaults should be suiting most needs. After you compiled it
+with make (or gmake if make doesn't work) you can install it with
+`make install` though you don't need to as ZNC supports in-place execution.
```

Comment:
Let's remove the though.... part. It kinda supports such execution, but has multiple flaws.